cli/cmd/generate.go

```diff
@@ -106,10 +107,9 @@ var (
 			}
 
 			// Prompt to execute
-			execute := false
 			err = SurveyQuestionInteractiveOnly(SurveyQuestionWithValidationArgs{
-				Prompt:   &survey.Confirm{Default: execute, Message: QuestionRunTfPlan},
-				Response: &execute,
+				Prompt:   &survey.Confirm{Default: GenerateAwsCommandExtraState.TerraformApply, Message: QuestionRunTfPlan},
```

Comment:
Wonder if the Message should be changed? `QuestionRunTfPlan` makes me think that we are just going to run the Plan action rather than Apply, which it seems will be the default for this now?

Reply:
So there are really two modes of operating here; one interactive and one non-interactive.

The question shown here is prompted only in interactive mode; so that's still OK (prompt for Plan execution, and then we prompt again for Apply execution).

If you pass `--apply` in interactive mode, all that happens is the default response in that prompt changes from N to Y, but you still (again, in interactive mode) have to go through a plan, review the results, and then confirm you want an apply to run.

If you pass `--apply` with `--non-interactive` then there is no option to run a plan, just the apply is run with `-auto-approve` flag set on the terraform command.  

So I think this one is OK to leave as is.  I'll resolve this, but we can unresolved this comment if there are concerns!